firstvoices/backend/serializers/async_results_serializers.py

```diff
@@ -88,3 +117,25 @@ class Meta:
             "latest_recalculation_preview_date",
             "latest_recalculation_preview_result",
         ]
+
+
+class BulkVisibilityJobSerializer(BaseJobSerializer):
+    url = SiteHyperlinkedIdentityField(
+        read_only=True, view_name="api:bulk-visibility-detail"
+    )
+    from_visibility = serializers.SerializerMethodField(read_only=True)
+    to_visibility = serializers.SerializerMethodField(read_only=True)
```

Comment:
You can reuse the EnumField here instead, and then it will work as a writable serializer as well

Reply:
Good catch!